platform_versioned_docs/version-24.1/monitoring/dashboard.mdx

```diff
@@ -25,5 +27,31 @@ Depending on the filter selected, each card details a separate workspace or orga
 
 For a detailed view of runs, you can do one of the following:
 
-- Select a run integer value in the table to navigate to a run list filtered by the status and time range selected.
-- Select a workspace name in the table to navigate to a run list filtered by the workspace selected.
+- Select a pipeline run integer value in the table to navigate to a pipeline run list filtered by the status and time range selected.
+- Select a workspace name in the table to navigate to a pipeline run list filtered by the workspace selected.
+
+## Data Studios
+
+You can explore the status of data studio sessions in your organizational workspaces. On the **Dashboard** page, select **Data Studios**.
+
+A summary of the number of data studio sessions in each status is available:
+
+- `Running`
+- `Errored`
+- `Starting`
+- `Stopping`
+- `Stopped`
+
+### Filters and summary
+
+The **Dashboard** view defaults to all organizations and workspaces you can access. Select the **View** dropdown menu to filter by organization(s) and workspace(s).
```

Comment:
@justinegeffen, I've never quite understood _(s)_. It is a bit self evident that you can filter by one or more, so it reads awkwardly whenever I see this construction in any documentation. It feels overly pedantic.

Reply:
I agree - and it was also inconsistent with the rest of the references. Thanks for catching it.